src/olympia/migrations/1087-abuse-add-guid-to-existing-addon-reports.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE `abuse_reports`
+    INNER JOIN `addons` ON (`abuse_reports`.`addon_id` = `addons`.`id`)
```

Comment:
for add-ons that have been deleted, but then replaced with a new add-on listing, `guid` will be a string like `guid-reused-by-pk-xxx` where x is the new add-on id.  Maybe that's fine just for backfilling, but it's not really the guid so just pointing it out.

Reply:
Yeah, I had that in mind - I think that's fine since it's just for backfilling.